src/contract/contract-type.checker.ts

```diff
@@ -105,6 +124,19 @@ export class ContractTypeChecker {
     }".`;
   }
 
+  private buildWrongRegexError(
+    propertyName: string,
+    parentPath: string | undefined,
+    contractProperty: Contract | ContractProperty,
+    valuesProperty: any,
+  ) {
+    const displayedPropertyName = this.buildParentPath(
+      parentPath,
+      propertyName,
+    );
+    return `Configuration property "${displayedPropertyName}" which equals "${valuesProperty}" does not match regex "${contractProperty.regex}" defined in contract`;
```

Comment:
`Configuration property "${displayedPropertyName}" with value "${valuesProperty}" does not match regex "${contractProperty.regex}"`